data/hai/hai reveal 1 intro.txt

```diff
@@ -693,14 +711,19 @@ mission "Hai Leaks Response 3"
 	on offer
 		conversation
 			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
-			`	You get down to business when Sayari inquires how your journey went.`
 			choice
+				`	Keep silent and let the meeting begin.`
+					goto "keep silent"
 				`	"I don't recall there being nearly so many pirates before."`
```

Comment:
I'm wondering where the player is in the situation. Are they already at the meeting? It seems like it in the next part, but there's no ques before that.

Reply:
There needs to be text mentioning you entered a meeting room.

Reply:
I have updated it so you are in a meeting room with a fancy holoprojector.
[Friction Diction~3025-05-29 just before leaks response 3.txt](https://github.com/endless-sky/endless-sky/files/9995633/Friction.Diction.3025-05-29.just.before.leaks.response.3.txt)